app/src/main/java/com/cornellappdev/android/eatery/CtownMenuActivity.java

```diff
@@ -115,10 +123,29 @@ public void onOffsetChanged(AppBarLayout appBarLayout, int verticalOffset) {
         mCafeLoc.setText(mCafeData.getBuildingLocation());
         mCafeImage = findViewById(R.id.ind_image);
         mCafeImage.setBackgroundColor(0xFFff0000);
-
+        mDollarSignOne = findViewById(R.id.dollar_sign_1);
+        mDollarSignTwo = findViewById(R.id.dollar_sign_2);
+        mDollarSignThree = findViewById(R.id.dollar_sign_3);
+        String price = ((CollegeTownModel) mCafeData).getPrice();
+        // price is either $, $$, or $$$. Adjust the corresponding texts
+        if (price.equals("$")) {
+            mDollarSignTwo.setTextColor(ContextCompat.getColor(this, R.color.inactive));
+            mDollarSignThree.setTextColor(ContextCompat.getColor(this, R.color.inactive));
+        } else if (price.equals("$$")) {
+            mDollarSignThree.setTextColor(ContextCompat.getColor(this, R.color.inactive));
+        } else {
```

Comment:
i don't think you need this else block... you can just have 
`if 
price.equals('$') {
} else { 
//code for price.equals('$$')
}`

Reply:
There's three price options, $, $$, and $$$. I can remove the last else block because nothing should be done if the price = $$$